docs/helm.md

```diff
@@ -64,3 +64,36 @@ spec:
 * `{{ .Values.resource.spec.name }}` would return "Nemo"
 * `{{ .Values.resource.spec.from }}` would return "Finding Nemo"
 * `{{ .Values.resource.spec.by }}` would return "Disney"
+
+### Using custom repo for charts
+
+For Lostrómos to be able to pull the charts from a remote repository, the remote
+repository should be pre-configured in the Lostrómos pod (one option is to add it
```

Comment:
This should state that it should be pre-configured in the tiller.

Reply:
done!